src/surveys/App.tsx

```diff
@@ -24,11 +34,11 @@ export default function App({ survey, onSurveyDisplayed, onSurveyClosed, onQuest
   };
 
   return (
-    <div id="fbjs">
+    <div id='fbjs'>
       <Wrapper
         // isOpen={isOpen}
-        // close={close}
```

Comment:
Do we still need the `isOpen` flag?

Reply:
It will be needed for the animation later.